adapters/airflow_connector.py

```diff
@@ -24,33 +25,60 @@ def query(self, request):
     Returns:
         The database result (tuple)
     """
-    cursor = self._airflow_db.cursor()
     # The following ensures that the query executes and returns,
     # even if the db connection has been lost
+    logging.info(request)
```

Comment:
When logging, you might want to give some more information. Just dumping the data is usually not really useful.